### ISSUE TITLE
Updated picture copy in activity feed is strange

### ISSUE THREAD

**Issue by anettlinno** [citizenos/citizenos-fe-old#1137]
**What is the user story?**

After changing public group picture, I received following notification in my "Activity feed":
<img width="428" alt="Pics" src="https://user-images.githubusercontent.com/45175679/206513590-cd1dd138-1ace-466b-92f1-b2d4f493c2c4.png">

**Which feature needs to be updated and how?**
I would rephrase it to "User Anett Linno has changed a photo in Group Citizen OS" or something like that :) OR only the "imageUrl" to "image" 

**Additional information.**
N/A

**Comment by ilmartyrk** [comment]
@anettlinno fixed, needs translations
